server/command.go

```diff
@@ -152,20 +166,16 @@ func (p *Plugin) executeCommandQueue(args *model.CommandArgs) *model.CommandResp
 		return responsef("Error calculating hashtags. Check the meeting settings for this channel.")
 	}
 
-	searchResults, appErr := p.API.SearchPostsInTeamForUser(args.TeamId, args.UserId, model.SearchParameter{Terms: &hashtag})
-
-	if appErr != nil {
-		return responsef("Error calculating list number")
+	numQueueItems, itemErr := calculateQueItemNumberAndUpdateOldItems(meeting, args, p, hashtag)
+	if itemErr != nil {
+		return itemErr
```

Comment:
```suggestion
		return errors.Wrap(err, "failed to requeue items after deleting an item")
```

Reply:
Actually, `itemErr` is a `CommandResponse` right here. That's concerning since this variable name implies that this is an `error`. 2/5 `calculateQueItemNumberAndUpdateOldItems` should be returning an `error` instead of a `CommandResponse`.

Reply:
Right, changed.